spring-components/rdf4j-spring/src/main/java/org/eclipse/rdf4j/spring/dao/support/RelationMapBuilder.java

```diff
@@ -55,6 +56,11 @@ public RelationMapBuilder(RDF4JTemplate rdf4JTemplate, RdfPredicate predicate) {
 		this.predicate = predicate;
 	}
 
+	public RelationMapBuilder(RDF4JTemplate rdf4JTemplate, IRI predicate) {
+		this.rdf4JTemplate = rdf4JTemplate;
+		this.predicate = iri(predicate);
+	}
+
```

Comment:
Are these changes required for the fix in TransactionObject?

Reply:
As long as the issues in the 4.1.0-M1 version are present, we desperately need these changes already in 4.0.4 for our project.